front/public/locales/fr/stdcm.json

```
@@ -135,16 +142,19 @@
       "timeOrigin": "l'heure de départ à l'origine",
       "totalLength": "la longueur totale du convoi",
       "totalMass": "le poids total du convoi",
-      "tractionEngine": "l'engin de traction"
+      "tractionEngine": "l'engin de traction",
+      "vias": "au moins un PR intermédiaire"
     },
     "missingInformations": "Informations manquantes pour le calcul :",
-    "missingLocation": "Localisation manquante",
     "noPaths": "Incompatibilité avec d'autres sillons.",
     "noResults": "Aucun sillon trouvé",
     "noScheduledPoint": "Le calcul a besoin de l'horaire d'origine ou de celui de la destination.",
     "pathfindingFailed": "Aucun chemin n'a été trouvé pour ces points de jalonnement.",
     "requestFailed": "Erreur de calcul Sillon de dernière minute",
-    "zeroLengthPath": "Le chemin ne peut pas avoir de longueur nulle."
+    "routeErrors": "Veuillez vérifier les erreurs liées au trajet suivantes :",
```

nit : less confusing like this maybe ?
```suggestion
    "routeErrors": "Veuillez vérifier les erreurs suivantes liées au trajet :",
```

---

Indeed.
Changed